Index: src/main/java/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>module com.example.hospitaldeliveryinterface {\r\n    requires javafx.controls;\r\n    requires javafx.fxml;\r\n\r\n\r\n    opens com.example.hospitaldeliveryinterface to javafx.fxml;\r\n    exports com.example.hospitaldeliveryinterface;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/module-info.java b/src/main/java/module-info.java
--- a/src/main/java/module-info.java	
+++ b/src/main/java/module-info.java	
@@ -1,7 +1,7 @@
 module com.example.hospitaldeliveryinterface {
     requires javafx.controls;
     requires javafx.fxml;
-
+    //requires fontawesomefx;
 
     opens com.example.hospitaldeliveryinterface to javafx.fxml;
     exports com.example.hospitaldeliveryinterface;
Index: src/main/java/com/example/hospitaldeliveryinterface/PharmaTracApp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.hospitaldeliveryinterface;\r\n\r\nimport javafx.application.Application;\r\nimport javafx.fxml.FXMLLoader;\r\nimport javafx.scene.Scene;\r\nimport javafx.stage.Stage;\r\n\r\nimport java.io.IOException;\r\n\r\npublic class PharmaTracApp extends Application {\r\n\r\n    private static Scene scene;\r\n    private static Stage stage;\r\n\r\n    @Override\r\n    public void start(Stage theStage) throws IOException {\r\n\r\n        stage = theStage;\r\n        FXMLLoader fxmlLoader = new FXMLLoader(PharmaTracApp.class.getResource(\"Homepage.fxml\"));\r\n        scene = new Scene(fxmlLoader.load(),800,600);\r\n        stage.setTitle(\"PharmaTrac\");\r\n        stage.setScene(scene);\r\n        stage.show();\r\n    }\r\n\r\n    static Scene getScene(){\r\n        return scene;\r\n    }\r\n\r\n    static Stage getStage(){\r\n        return stage;\r\n    }\r\n\r\n    public static void main(String[] args) {\r\n        launch();\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/hospitaldeliveryinterface/PharmaTracApp.java b/src/main/java/com/example/hospitaldeliveryinterface/PharmaTracApp.java
--- a/src/main/java/com/example/hospitaldeliveryinterface/PharmaTracApp.java	
+++ b/src/main/java/com/example/hospitaldeliveryinterface/PharmaTracApp.java	
@@ -32,6 +32,7 @@
     }
 
     public static void main(String[] args) {
+
         launch();
     }
 }
\ No newline at end of file
